dev/Repeater/VirtualizingLayout.h

```diff
@@ -10,6 +10,7 @@ class VirtualizingLayout :
     public winrt::implementation::VirtualizingLayoutT<VirtualizingLayout, Layout>
 {
 public:
+    VirtualizingLayout();
```

Comment:
should we also add the telemetry for flowlayout ?

Reply:
We certainly can.  The original request was for these four layouts.

Reply:
Consulted with Premalini and she suggested leaving it at these four.  If you think that we should add flowlayout, re-activate this conversation and I'll make the changes.